### PR TITLE
Added more links to videos at Aurora

### PR DESCRIPTION
**Pull request by Krabby127**
Found more videos of what happened at Aurora and these seem to show the genesis moments of when violence starts.

### CODE REVIEW

**Comment by Krabby127** [comment]
Am I not doing something right? I removed a tag I added that didn't fit and now my fork is building but the master isn't apparently.

**Comment by emarcey** [comment]
The validation doesn't work with branches from remote repos just yet. Sorry for the confusion. 

But you can remove the yml changes, I think

File: reports/Colorado.md [hunk 1]
```diff
@@ -6,15 +6,16 @@ On June 27th, citizens of Aurora held a violin vigil in honor of Elijah McClain.
...
-tags: strike, beat, baton, pepper-spray, spray
+tags: strike, beat, baton, pepper-spray, spray, tear-gas
 
 id: co-aurora-1
 
 **Links**
 
 * https://www.reddit.com/r/Bad_Cop_No_Donut/comments/hhfrh8/062820_musicians_in_aurora_colorado_honoring/
 * https://www.reddit.com/r/2020PoliceBrutality/comments/hhh6fe/aurora_co_during_a_vigil_with_live_violinists_for/
-* https://twitter.com/jessiedesigngir/status/1277126019246260224
```
Comment (emarcey) [inline reports/Colorado.md:17]: Why did you remove this link? It shows the police arriving on scene

File: reports/Colorado.md [hunk 2]
```diff
@@ -6,15 +6,16 @@ On June 27th, citizens of Aurora held a violin vigil in honor of Elijah McClain.
 
 Aurora police arrived in riot gear to disperse the vigil. When participants refused, riot police beat them with batons and pepper-sprayed them.
 
-tags: strike, beat, baton, pepper-spray, spray
+tags: strike, beat, baton, pepper-spray, spray, tear-gas
```
Comment (emarcey) [inline reports/Colorado.md:9]: I'm not sure about the tear gas. It may have just been smoke grenades, so unless there are recovered canisters or something, I would prefer not to make the assumption

File: reports/Colorado.md (hunk 1 above)
Comment (Krabby127) [reply reports/Colorado.md:17]: I thought that link was already in one of the above reddit posts, but I was mistaken

File: reports/Colorado.md (hunk 2 above)
Comment (Krabby127) [reply reports/Colorado.md:9]: That's a good point. Aurora PD said they **didn't** use tear-gas, but that smoking canister being tossed is suspicious.

Comment (emarcey) [reply reports/Colorado.md:9]: Yeah but it could easily just be a smoke grenade. So I think you could include the ambiguity in the description, but I would remove the tag.

Comment (Krabby127) [reply reports/Colorado.md]: removed the tag

File: reports/Colorado.md (hunk 1 above)
Comment (Krabby127) [reply reports/Colorado.md:17]: Added link back
